setup.py

```diff
@@ -120,11 +120,23 @@ def get_git_sha():
         "elasticsearch": ["elasticsearch-dbapi>=0.1.0, <0.2.0"],
         "druid": ["pydruid>=0.6.1,<0.7"],
         "hana": ["hdbcli==2.4.162", "sqlalchemy_hana==0.4.0"],
-        "dremio": ["sqlalchemy_dremio>=1.1.0"],
-        "cockroachdb": ["cockroachdb==0.3.3"],
+        "dremio": ["sqlalchemy-dremio>=1.1.5, <1.2"],
```

Comment:
Nit @blcksrx would you mind sorting these in alphabetical order? It's easier to grok which engines are supported using this approach.